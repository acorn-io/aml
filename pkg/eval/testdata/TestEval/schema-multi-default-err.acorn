schema {
    foo: "hi"
    foo: "bye"
}

foo: "huh"

