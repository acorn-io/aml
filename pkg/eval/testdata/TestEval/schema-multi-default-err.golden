"multiple defaults can not be defined: schema-multi-default-err.acorn:6:1 (3:5<-1:8<-6:1)"
